addons/vehicles/CfgVehicles/Raptor.inc

```diff
@@ -0,0 +1,144 @@
+
+#undef AUTHOR
+#define AUTHOR Demon
```

Comment:
This will cause issues depending on when things are loaded. Instead, in script_component.hpp just add yourself to the AUTHORS array; I'm probably going to change AUTHOR to be something like 'SWS DevTeam' or similar soon anyway.

Reply:
I thought that might be an issue.